share/clus_apis.go

```diff
@@ -1111,6 +1111,8 @@ type CLUSGroupMetric struct {
 type CLUSWlMetric struct {
 	WlID        string `json:"wlid"`
 	WlSessCurIn uint32 `json:"wl_sess_cur_in"`
+	WlSessIn1   uint32 `json:"wl_sess_in1"`
+	WlByteIn1   uint64 `json:"wl_byte_in1"`
```

Comment:
Why these 2 lines are in this PR? Are they compliance-related?

Reply:
Thanks for pointing out.